### ISSUE TITLE
 export house preview video sync issue

### ISSUE THREAD

**Issue by JeffHaberman** [xLightsSequencer/xLights#1256]
I've created some musical sequences with one using a custom 30 ms timing and one with 25 ms timing.  When I export them to mp4 files using the "export house preview video" feature the audio is out of sync with the the lights.  Have you seen this issue before?  Thanks

**Comment by KMojek** [comment]
Actually, I didn't realize xLights had an option for custom timing, so no that hasn't been tested. I will see if I can reproduce the issue.

**Comment by jholmes09** [comment]
Yes I’ve had this issue multiple items. What I do as a workaround is set the frame rate to the standard 40 FPS or 20 FPS, export the video, then set my sequence back to the custom rate I want.

**Comment by derwin12** [comment]
I cant duplicate this and was likely addressed with the updates and rewrites to the ffmpeg house export view.
